src/App.tsx

```diff
@@ -1,6 +1,12 @@
 import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client'
 import MapView from '@components/MapView'
+//import SearchView from '@components/SearchView'
+//import BusinessView from '@components/BusinessView'
```

Comment:
Remove commented imports.

Reply:
I was doing this to make it easy to switch between views for testing and easy to add each view once we implement navigation.